apmproxy/apmserver.go

```diff
@@ -30,11 +31,21 @@ import (
 	"time"
 )
 
+type jsonResult struct {
+	Accepted int         `json:"accepted"`
```

Comment:
Seeing as we don't use this field, should we remove it from the struct to avoid decoding it?

Reply:
Good catch!